src/pages/Country.module.scss

```diff
@@ -35,19 +37,51 @@
   padding: 20px;
   border-radius: 3px;
   cursor: pointer;
+  transition: all 0.2s;
 }
 
 .buttonBlock {
   @include theme($theme: $veryDarkBlue);
 
+  width: 100%;
   display: flex;
   box-shadow: none;
+  flex-wrap: wrap;
+  transition: all 0.2s;
 }
 
 .link {
   all: unset;
 }
 
+@media ($tablet) {
+  .cardImg {
+    width: 35%;
+    margin-right: 70px;
+  }
+
+  .cardBlock {
+    height: calc(100vh - 400px);
```

Comment:
Why `400px` exactly?

Reply:
To be honest, I worked by eye :)
I don’t know if it’s correct, but now I made the main block minus the size of the header

Reply:
I mean it's not obvious if it's a header or any other element on a page.

I suggest creating custom css variable and use it here. Yes, it will be only in one place.

Or you can just add comment to this line to not forget what 400px means.

Reply:
done